### ISSUE TITLE
BlockUI always enabled just by importing the library

### ISSUE THREAD

**Issue by ggam** [TheCoder4eu/BootsFaces-OSP#790]
Manually importing the BlockUI library enables it for AJAX requests even with it being disabled on the `web.xml`. BootsFaces AJAX API checks for the presence of the `$.blockUI` and if it's present and not null, it automatically blocks and unblocks the screen: https://github.com/TheCoder4eu/BootsFaces-OSP/blob/5bdbf2288c4cfa06a3c53d1226259e45bfe1708a/gradleResources/staticResources/js/bsf.js#L39

Since BlockUI cannot be disabled on a per request basis (like PrimeFaces does with the `global` attribute), I imported the library and disabled it on `web.xml`. But BootsFaces doesn't do the check on the client side.

I propose two changes here:
- [bug]: importing the library should not automatically enable it
- [feature]: a new way to disable BlockUI for a specific component/event (like PrimeFaces).

The workaround I've used for now is to rename the variables on my imported JS file, downloaded from the author's website.

**Comment by stephanrauh** [comment]
I guess it's easy to implement the check of the web.xml setting. If I'm not mistaken, you've already done similar things. If you want to submit the PR yourself, go ahead!

**Comment by ggam** [comment]
I guess just adding a EL expression to the JS file should work, no?
Something like:

if($.BlockUI != null && #{blockUIEnabled}) {
  // ...
}

The resource mechanism should interpret it. Would that be enough? I'l take
a look and try to provide a PR when I find some time.

**Comment by chongma** [comment]
slightly related: i was doing pagespeed insights https://developers.google.com/speed/pagespeed/insights/ the other day and it said that the BlockUI javascript file was not minified.  does BootsFaces use the minified version?

**Comment by stephanrauh** [comment]
Actually, there's no official minified version of BlockUI. I minified it manually.

**Comment by stephanrauh** [comment]
We've implemented this feature long ago. I've just added the documentation, so we can close the ticket.
